.SRCINFO

```diff
@@ -1,31 +1,50 @@
 pkgbase = amdgpu-pro-installer
-	pkgver = 17.10.401251
+	pkgver = 17.30.465504
 	pkgrel = 2
 	url = http://www.amd.com
 	arch = x86_64
 	license = custom:AMD
 	makedepends = wget
-	source = https://www2.ati.com/drivers/linux/ubuntu/amdgpu-pro-17.10-401251.tar.xz
+	source = https://www2.ati.com/drivers/linux/ubuntu/amdgpu-pro-17.30-465504.tar.xz
 	source = 0001-disable-firmware-copy.patch
-	source = 0002-linux-4.9-fixes.patch
-	source = 0003-Change-seq_printf-format-for-64-bit-context.patch
-	source = 0004-fix-warnings-for-Werror.patch
-	source = 0005-add-archlinux-as-build-option.patch
-	sha256sums = 0a10cf39841bf77eacb393ca112ce5f82ca0c4ea728d2fce975732855c039600
-	sha256sums = 6aaf7566a70d1654bfc1ddfc73f25b246518e39c504c350cd131f4c40ce5a583
-	sha256sums = 858828f9292e552dc2a20d838c602ab76e56c32384352d3de26e519c18a16db5
-	sha256sums = 1e22be9d3e3aed6848cc0030f37d3fe710b37322d6db2219521827edf02cf38c
-	sha256sums = 375a0caad148e018774fae57cc1c05926b01b71497c363b38d83163b0e6961ca
-	sha256sums = 4e29fd4c1fd319b22c42d4d9e26e5cc75b9eab61e15f65e8acbe6db8a31ca3fb
+	source = 0002-fix-warnings-for-Werror.patch
+	source = 0003-add-archlinux-as-build-option.patch
+	sha256sums = b74494d142542164b5582e8bed845315f788aa3d9eb0c1ed5f2a6c5f982beaf5
+	sha256sums = c9b95504f5f7575c7ca7e138beb4bf474efd2abe82e72e9cad3d4fd1a8fc5c7c
+	sha256sums = 4d63683a846b16ca914557e371df90460e45639a2a7ca0df0cf575b945a4661a
+	sha256sums = 2bf8eac2afac3bce0d17f840c15009838b578a72412ff66df6e8caa6c00fade7
 
 pkgname = amdgpu-pro
 	pkgdesc = The AMDGPU Pro driver package
 	install = amdgpu-pro-core.install
 	arch = x86_64
+	depends = binfmt-support
 	depends = glib2>=2.37.3
 	depends = gst-plugins-base>=1.6.0
 	depends = gstreamer>=1.0.0
+	depends = hsa-ext-amdgpu-pro-finalize=17.30.465504-2
+	depends = hsa-ext-amdgpu-pro-image=17.30.465504-2
+	depends = hsa-runtime-tools-amdgpu-pro=17.30.465504-2
+	depends = lib32-amdgpu-pro-core=17.30.465504-2
+	depends = lib32-binfmt-support
+	depends = lib32-libedit2>=2.11-20080614
+	depends = lib32-libffi-dev>=3.0.9
+	depends = lib32-libllvm5.0-amdgpu-pro=17.30.465504-2
+	depends = lib32-libtinfo-dev
+	depends = lib32-llvm-amdgpu-pro-5.0-dev=17.30.465504-2
+	depends = lib32-llvm-amdgpu-pro-5.0-runtime=17.30.465504-2
+	depends = lib32-llvm-amdgpu-pro-5.0=17.30.465504-2
+	depends = lib32-llvm-amdgpu-pro-runtime=17.30.465504-2
+	depends = lib32-ncurses5-compat-libs>=6
+	depends = lib32-zlib>=1.2.0
+	depends = libedit2>=2.11-20080614
+	depends = libffi-dev>=3.0.9
+	depends = libjs-jquery
+	depends = libjs-underscorea
```

Comment:
Typo? (libjs-underscore)

Reply:
Not a direct typo (these are generated from the .debs), but it does seem wrong.